keyboards/ymdk/ymd75/readme.md

```diff
@@ -4,21 +4,10 @@
 
 This firmware can also be flashed onto the KBDFans KBD75 Round 2 Bootmapper Client boards sold during the mid-2018 timeframe.
```

Comment:
Looking at https://github.com/qmk/qmk_firmware/commit/5660d80bb01aa4240a5b11d2d60568e21093817a, I suppose that this part applies only to `ymd75/rev1`.

Reply:
cb19e28c5419a7c9938141e3da55911c87e6f16b should hopefully cover this